cashews/backends/memory.py

```diff
@@ -72,6 +72,10 @@ async def get_raw(self, key: str):
     async def get_many(self, *keys: str, default: Optional[Any] = None) -> Tuple[Any, ...]:
         return tuple(self._get(key, default=default) for key in keys)
 
+    async def set_many(self, data: Dict[str, Any], expire: Optional[float] = None) -> bool:
```

Comment:
You annotated that we expect `bool` as result  but you do not return any (only `None`).
Also the [main interface](https://github.com/Krukov/cashews/pull/88/files#diff-3a4cf252cb2da2c2cc4fad84cfad4edf04d871a0bfacb635cfd343ff306f0533R44) have different signature - have nothing to return (void)